match2/wikis/starcraft2/match_group_legacy.lua

```diff
@@ -59,9 +59,6 @@ function Legacy.get(frame)
 	newArgs.store = storage
 	newArgs.noDuplicateCheck = _args.noDuplicateCheck
 
-	newArgs.store = storage
-	newArgs.noDuplicateCheck = _args.noDuplicateCheck
-
```

Comment:
Is this deletion right?

Reply:
it was twice there so just removing redundant duplicate code

![Screenshot 2021-07-19 09 45 46](https://user-images.githubusercontent.com/75081997/126122305-67633728-754e-426d-9363-75c04a427e1b.png)